inginious/frontend/i18n/fr/LC_MESSAGES/messages.po

```diff
@@ -1572,32 +1586,64 @@ msgstr ""
 msgid "Name"
 msgstr "Nom"
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:69
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:67
+msgid "The name field is required"
+msgstr ""
+
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:75
 #: inginious/frontend/plugins/user_management/pages/templates/user_management.html:27
 #: inginious/frontend/templates/course_admin/edit_tabs/files.html:17
 msgid "Edit"
 msgstr "Éditer"
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:75
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:81
 msgid "Course Name"
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:81
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:85
+#: inginious/frontend/plugins/course_creation/templates/create_course_modal.html:29
+#: inginious/frontend/templates/task.html:133
+msgid "Group"
+msgstr "Groupe"
+
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:91
 msgid "Comments or description"
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:82
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:92
 msgid "Put your opinion or tell us about the problem you have encountered "
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:83
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:93
 msgid "Describe and justify the course you want to create "
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:89
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:95
+msgid "The comment is too short"
+msgstr ""
+
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:100
 msgid "Send"
 msgstr ""
 
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:108
+#, fuzzy
+msgid "Message Status"
+msgstr "Etat du service"
```

Comment:
As in the last PR, delete fuzzy labels and messages that were automatically translated.